lib/browser/chrome-extension.js

```diff
@@ -322,7 +328,7 @@ app.once('ready', function () {
     const manifest = getManifestFromPath(srcDirectory)
     if (manifest) {
       for (const webContents of getAllWebContents()) {
-        if (webContents.getType() !== 'remote') {
+        if (isWindowOrWebView(webContents)) {
```

Comment:
Can you explain why [BrowserView](https://electronjs.org/docs/api/browser-view) is not supported? @kevinsawicki  @zcbenz